app/pages/sections/catalogue-solutions/order-item/contextCreator.js

```diff
@@ -0,0 +1,7 @@
+import manifest from './manifest.json';
+
+export const getContext = ({ solutionName, serviceRecipientName, odsCode }) => ({
+  ...manifest,
+  title: `${solutionName} info for ${serviceRecipientName} (${odsCode})`,
```

Comment:
Can we add this `info for` to the manifest. Just so we know all our content comes from there.